src/api/journalEntries/journalEntry.ts

```diff
@@ -117,15 +117,16 @@ interface JournalEntryUpdate {
   garlandPose: number | string;
   prenatalVitamins: boolean | null;
   probiotics: boolean | null;
+  authorId: string | undefined;
 }
 
 const updateJournalEntryMutation = gql`
   mutation updateJournalEntry(
-    $data: JournalEntryUpdateInput!
-    $where: JournalEntryWhereUniqueInput!
+    $data: JournalEntryCreateInputData!
+    $updateJournalEntryId: Int!
```

Comment:
id is not put into the data so we won't be resetting it because it doesn't change

Reply:
I feel like authorId should be included like this